lib/ensure/symlink-paths.js

```diff
@@ -1,6 +1,8 @@
-var path = require('path')
+'use strict'
+
+const path = require('path')
 // path.isAbsolute shim for Node.js 0.10 support
```

Comment:
Sort of unrelated, but this comment should have been removed earlier, wouldn't hurt to remove it here.